### DISCUSSION TITLE
Intersection Observer v2 delay and trackVisibility properties missing in Browser compatibility and are not described - plus no support in Firefox?

### DISCUSSION THREAD

**Discussion by BearCooder** [mdn/mdn-community#169] [Content]
I cannot find information about the Intersection Observer v2 on MDN if it is supported - especially the properties `delay` and `trackVisibility`. A search with different keywords on google for mdn and in the mdn searchbox shows nothing. And in all the Browser compatibility boxes connected with Intersection Observer I did not find those two properties.

_The IntersectionObserver constructor now takes two additional configuration properties: delay and trackVisibility. The delay is a number indicating the minimum delay in milliseconds between notifications from the observer for a given target. The trackVisibility is a boolean indicating whether the observer will track changes in a target's visibility._ [Source](https://web.dev/intersectionobserver-v2/) and [spec](https://w3c.github.io/IntersectionObserver/v2/#calculate-visibility-algo)

I did a quick code test and it seems that Firefox does really not support these two properties. It worked on Chrome but on Firefox not.

**Comment by teoli2003** [comment]
I searched Bugzilla and found nothing. I think we can safely assume this is Chrome-only for the time being.

**Comment by BearCooder** [comment]
sadface :(
But it would be still good to include in the MDN documentation + Browser support table I think? :)

**Reply by teoli2003** [reply]
If it is supported by a browser (Chrome, for example), of course. PR(s) welcome.
